recipes/kmer-jellyfish/1.1.12/meta.yaml

```diff
@@ -34,3 +37,5 @@ about:
 extra:
   identifiers:
     - biotools:jellyfish
+  additional-platforms:
```

Comment:
nit: I try to keep the list items sorted by name, i.e. I'd move `additional-platforms` before `identifiers`